tools/pubsys/src/repo.rs

```diff
@@ -102,6 +102,12 @@ pub(crate) struct RepoArgs {
     outdir: PathBuf,
 }
 
+pub(crate) fn root_bytes(path: impl AsRef<Path>) -> Result<Vec<u8>> {
+    std::fs::read(path.as_ref()).context(error::FileSnafu {
```

Comment:
Is there a reason not to use an async read?

```suggestion
    fs::read(path.as_ref()).context(error::FileSnafu {
```